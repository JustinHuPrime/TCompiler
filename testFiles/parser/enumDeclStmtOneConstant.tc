module foo;

void bar() {
  enum e {
    A
  };
}