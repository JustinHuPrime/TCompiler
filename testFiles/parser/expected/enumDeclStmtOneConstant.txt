testFiles/parser/input/enumDeclStmtOneConstant.tc (code):
FILE(1, 1, STAB(ENTRY(bar, FUNCTION(testFiles/parser/input/enumDeclStmtOneConstant.tc, 3, 1, void()))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), FUNDEFN(3, 1, KEYWORDTYPE(3, 1, void), ID(3, 6, bar, REFERENCES(testFiles/parser/input/enumDeclStmtOneConstant.tc, 3, 1)), STAB(), COMPOUNDSTMT(3, 12, STAB(ENTRY(e, ENUM(testFiles/parser/input/enumDeclStmtOneConstant.tc, 4, 3, CONSTANT(A, 0)))), ENUMDECL(4, 3, ID(4, 8, e, REFERENCES(testFiles/parser/input/enumDeclStmtOneConstant.tc, 4, 3)), ID(5, 5, A, REFERENCES(testFiles/parser/input/enumDeclStmtOneConstant.tc, 5, 5)), (null)))))
